lib/iris/tests/__init__.py

```diff
@@ -91,6 +92,13 @@
     GRIB_AVAILABLE = True
 
 
+#: CDL history attribute regular expression pattern for ISO8601 timestamp.
+iso8601 = '(?P<lhs>\s*:history = ")' \
+    '(?P<timestamp>\d{4}\-?\d{2}\-?\d{2}' \
+    '(T\d{2}\:?\d{2}\:?\d{2}(\.\d{3})?(([\+\-]\d{2}\:?\d{2})|Z)?)?: )' \
+    '(?P<rhs>.*)'
```

Comment:
I'd love to be able to do that (as much fun as it was blowing my tubes out with re) ... but I don't quite know how, hence re carnage :facepunch: :boom: :feelsgood: